src/main/java/org/jenkinsci/plugins/credentialsbinding/impl/ZipFileBinding.java

```diff
@@ -53,6 +54,7 @@
         secret.unzipFrom(credentials.getContent());
     }
 
+    @Symbol("zip")
```

Comment:
I do not know of a `zip` step offhand. If there is one, _Snippet Generator_ will automatically avoid using the symbol.